### DISCUSSION TITLE
Timeout between 2 recuring transaction run

### DISCUSSION THREAD

**Discussion by Old-Veeh** [firefly-iii/firefly-iii#6060] [Firefly III]
Hi,

I'm running Firefly in docker and I have 2 accounts.
I have set a cron to trigger the recurring transaction every day, one for each account, and I'm using the API URL.
But only one account's transactions are going through, the first to go.

First account all good.
    "output.stdout": {
        "auto_budgets": {
            "job_errored": false,
            "job_fired": true,
            "job_succeeded": true,
            "message": "Auto-budget cron job fired successfully."
        },
        "recurring_transactions": {
            "job_errored": false,
            "job_fired": true,
            "job_succeeded": true,
            "message": "Recurring transactions cron job fired successfully."
        }

Second account
    "output.stdout": {
        "auto_budgets": {
            "job_errored": false,
            "job_fired": false,
            "job_succeeded": false,
            "message": "It has been 1s after since the auto budget cron-job has fired. It will not fire now."
        },
        "recurring_transactions": {
            "job_errored": false,
            "job_fired": false,
            "job_succeeded": false,
            "message": "It has been 1s after since the recurring transactions cron-job has fired. It will not fire now."
        }

So my question is, is there a minimum timeout to respect between each run ?
Thank you

**Comment by JC5** [comment]
What do you mean with "account"?

**Comment by Old-Veeh** [comment]
I meant user.
Both I and my wife have a user profile in the app. And we both have our token to trigger the recursive transactions.

**Reply by JC5** [reply]
The recurring cron job should fire for both users, that's how it's built. The debug logs will reflect this. It's not necessary to set up  a cron job for the second user.

**Reply by Old-Veeh** [reply]
oh ok. I did not get that.
Thanks for the clarification.
Have a nice day
